send_to_teams.py

```diff
@@ -0,0 +1,92 @@
+import requests
+import logging
+
+from PIL import Image
+from io import BytesIO
+
+from content_loader import ContentLoader
+import credentials
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class TeamsNotifier:
+    def __init__(self, quote_loader: ContentLoader, image_loader: ContentLoader):
+        self.quote_loader = quote_loader
+        self.image_loader = image_loader
```

Comment:
Actually, I think that it would be better to have in this class not a logic of fetching your data, but a logic of just sending it. The rest of your work should be done in main script